### ISSUE TITLE
segmentation fault

### ISSUE THREAD

**Issue by 1808030112** [Janspiry/Palette-Image-to-Image-Diffusion-Models#92]
This is my train.log,When I start running and the progress bar shows 0%, I will report this segment error. This problem has been bothering me for a long time, but I don't know how to solve it.
The environment was installed according to requirements. txt, perhaps because my Python version is incorrect?

23-12-27 10:15:08.711 - INFO: Create the log file in directory experiments/train_inpainting_infared_231227_101508.

23-12-27 10:15:08.737 - INFO: Dataset [InpaintDataset() form data.dataset] is created.
23-12-27 10:15:08.738 - INFO: Dataset for train have 8 samples.
23-12-27 10:15:08.738 - INFO: Dataset for val have 2 samples.
23-12-27 10:15:09.234 - INFO: Network [Network() form models.network] is created.
23-12-27 10:15:09.235 - INFO: Network [Network] weights initialize using [kaiming] method.
23-12-27 10:15:09.622 - WARNING: Config is a str, converts to a dict {'name': 'mae'}
23-12-27 10:15:09.903 - INFO: Metric [mae() form models.metric] is created.
23-12-27 10:15:09.904 - WARNING: Config is a str, converts to a dict {'name': 'mse_loss'}
23-12-27 10:15:09.904 - INFO: Loss [mse_loss() form models.loss] is created.
23-12-27 10:15:12.613 - INFO: Beign loading pretrained model [Network] ...
23-12-27 10:15:12.613 - INFO: Loading pretrained model from [checkpoint/200_Network.pth] ...
23-12-27 10:15:12.860 - INFO: Beign loading pretrained model [Network_ema] ...
23-12-27 10:15:12.860 - WARNING: Pretrained model in [checkpoint/200_Network_ema.pth] is not existed, Skip it
23-12-27 10:15:12.862 - INFO: Beign loading training states
23-12-27 10:15:12.862 - INFO: Loading training state for [checkpoint/200.state] ...
23-12-27 10:15:13.297 - INFO: Model [Palette() form models.model] is created.
23-12-27 10:15:13.297 - INFO: Begin model train.

**Comment by 1808030112** [comment]
export CUDA_VISIBLE_DEVICES=0
run.py:29: UserWarning: You have chosen to use cudnn for accleration. torch.backends.cudnn.enabled=True
  warnings.warn('You have chosen to use cudnn for accleration. torch.backends.cudnn.enabled=True')
  0%|                                                                    | 0/2 [00:00<?, ?it/s]
段错误
